src/Microsoft.Health.Fhir.Shared.Core/Extensions/ModelExtensions.cs

```diff
@@ -28,10 +29,27 @@ public static OperationOutcome.IssueComponent ToPoco(this OperationOutcomeIssue
         {
             EnsureArg.IsNotNull(issue, nameof(issue));
 
+            CodeableConcept details = null;
+            if (issue.DetailsCodes != null && issue.DetailsText != null)
```

Comment:
IsNullOrEmpty()?
Also if both are required should we have model encapsulation? e.g. `OperationOutcomeIssueDetails`?

Reply:
Both of them optional, so I rewrote if statement.

---
In reply to: [574165776](https://github.com/microsoft/fhir-server/pull/1654#discussion_r574165776) [](ancestors = 574165776)